permissions/helpers.py

```diff
@@ -252,12 +252,10 @@ def get_units_where_can_view_reservations(user: User) -> list:
     if has_general_permission(user, permission) or is_superuser(user):
         return list(Unit.objects.all())
 
-    return list(
-        map(
-            lambda role: role.unit,
-            user.unit_roles.filter(role__permissions__permission=permission),
-        )
-    )
+    units = user.unit_roles.filter(
```

Comment:
This is a great change! :+1: 

But, if we would have a type checker, this would not pass.

Function states that it will return a list, but this is a queryset. Can you update this to execute the query or update the return type to a union (`Union[list, queryset]` or whatever queryset should be annotated as :smile: )?